pulsar-broker/src/main/java/org/apache/pulsar/broker/lookup/v2/TopicLookup.java

```diff
@@ -72,6 +80,10 @@ public void lookupTopicAsync(
     @GET
     @Path("{topic-domain}/{tenant}/{namespace}/{topic}/bundle")
     @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(
+            value = "Get the namespace bundle which contains the given topic.",
```

Comment:
```suggestion
            value = "Get the namespace bundle which the given topic belongs to.",
```

Reply:
Agree, Maybe also need to change it?

https://github.com/apache/pulsar/blob/81841eba724e41d243ba7afc58fcc7634513938b/site2/docs/reference-pulsar-admin.md?plain=1#L2664-L2665

Reply:
This doc page is deprecated and no longer maintained.
You can find the related information on [this page](https://pulsar.apache.org/docs/next/admin-api-topics/#get-bundle).